File: code/bb039.cpp
#include <iostream>
using namespace std;
int main()
{
    int n, gio, phut, giay;
    cin>>n;
    phut=n/60;
    giay=n%60;
    gio=phut/60;
    phut=phut%60;
    cout<<gio<<":";
    if (phut<10) cout<<0;
    cout<<phut<<":";
    if (giay<10) cout<<0;
    cout<<giay;
    return 0;
}

File: code/bb039.pas
var n, phut, giay, gio  : word;
begin
    readln(n);
    phut:= n div 60;
    giay:=n mod 60;
    gio:=phut div 60;
    phut:= phut mod 60;
    write(gio);
    if phut<10 then write(0);
    write(phut);
    if giay<10 then write(0);
    write(giay);
end.
